static/app/views/dashboards/widgets/lineChartWidget/lineChartWidget.spec.tsx

```diff
@@ -70,6 +73,7 @@ describe('LineChartWidget', () => {
       expect(
         screen.getByText(/does not contain any plottable values/)
       ).toBeInTheDocument();
+      jest.resetAllMocks();
```

Comment:
Perhaps it makes more sense to resetAllMocks in a `beforeEach`?

Reply:
I thought about it, but here only one test uses it 🤔 On the other hand, `afterEach` will run even if the test crashes for some reason, so it's good practice. Moved, thanks!